source/MultiGrid/MultiGrid.js

```diff
@@ -144,6 +159,23 @@ export default class MultiGrid extends PureComponent {
       this._topGridHeight = null
     }
 
+    if (
+      nextProps.scrollLeft !== this.state.scrollLeft ||
+      nextProps.scrollTop !== this.state.scrollTop
```

Comment:
This should compare `nextProps` to `this.props` instead of `this.state` right? Otherwise, it will interfere with user scrolling.

This kind of half-controlled prop is troublesome. In hindsight, I should have probably used a different approach here. Maybe a public method you call on the ref instead, I dunno.